Unit tests are failing

________ ERROR collecting tests/activity/test_activity_registration.py _________
tests/activity/test_activity_registration.py:2: in <module>
    from sorcha.activity.activity_registration import (
src/sorcha/__init__.py:1: in <module>
    from . import modules
src/sorcha/modules/__init__.py:16: in <module>
    from . import PPOutput
src/sorcha/modules/PPOutput.py:10: in <module>
    from tables import NaturalNameWarning
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/tables/__init__.py:44: in <module>
    from .utilsextension import get_hdf5_version as _get_hdf5_version
tables/utilsextension.pyx:1: in init tables.utilsextension
    ???
E   ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
________ ERROR collecting tests/ephemeris/test_ephemeris_generation.py _________
tests/ephemeris/test_ephemeris_generation.py:6: in <module>
    from sorcha.utilities.dataUtilitiesForTests import get_test_filepath, get_demo_filepath
src/sorcha/__init__.py:1: in <module>
    from . import modules
src/sorcha/modules/__init__.py:16: in <module>
    from . import PPOutput
src/sorcha/modules/PPOutput.py:10: in <module>
    from tables import NaturalNameWarning
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/tables/__init__.py:44: in <module>
    from .utilsextension import get_hdf5_version as _get_hdf5_version
tables/utilsextension.pyx:1: in init tables.utilsextension
    ???
E   ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
_______________ ERROR collecting tests/ephemeris/test_pixdict.py _______________
tests/ephemeris/test_pixdict.py:3: in <module>
    from sorcha.utilities.dataUtilitiesForTests import get_test_filepath, get_demo_filepath
src/sorcha/__init__.py:1: in <module>
    from . import modules
src/sorcha/modules/__init__.py:16: in <module>
    from . import PPOutput
src/sorcha/modules/PPOutput.py:10: in <module>
    from tables import NaturalNameWarning
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/tables/__init__.py:44: in <module>
    from .utilsextension import get_hdf5_version as _get_hdf5_version
tables/utilsextension.pyx:1: in init tables.utilsextension
    ???
E   ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
__________ ERROR collecting tests/ephemeris/test_simulation_driver.py __________
tests/ephemeris/test_simulation_driver.py:4: in <module>
    from sorcha.ephemeris.simulation_driver import calculate_rates_and_geometry, EphemerisGeometryParameters
src/sorcha/__init__.py:1: in <module>
    from . import modules
src/sorcha/modules/__init__.py:16: in <module>
    from . import PPOutput
src/sorcha/modules/PPOutput.py:10: in <module>
    from tables import NaturalNameWarning
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/tables/__init__.py:44: in <module>
    from .utilsextension import get_hdf5_version as _get_hdf5_version
tables/utilsextension.pyx:1: in init tables.utilsextension
    ???
E   ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
_________ ERROR collecting tests/ephemeris/test_simulation_parsing.py __________
tests/ephemeris/test_simulation_parsing.py:3: in <module>
    import sorcha.ephemeris.simulation_parsing as sp
src/sorcha/__init__.py:1: in <module>
    from . import modules
src/sorcha/modules/__init__.py:16: in <module>
    from . import PPOutput
src/sorcha/modules/PPOutput.py:10: in <module>
    from tables import NaturalNameWarning
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/tables/__init__.py:44: in <module>
    from .utilsextension import get_hdf5_version as _get_hdf5_version
tables/utilsextension.pyx:1: in init tables.utilsextension
    ???
E   ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
_______________ ERROR collecting tests/readers/test_CSVReader.py _______________
tests/readers/test_CSVReader.py:7: in <module>
    from sorcha.readers.CSVReader import CSVDataReader
src/sorcha/__init__.py:1: in <module>
    from . import modules
src/sorcha/modules/__init__.py:16: in <module>
    from . import PPOutput
src/sorcha/modules/PPOutput.py:10: in <module>
    from tables import NaturalNameWarning
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/tables/__init__.py:44: in <module>
    from sorcha.utilities.dataUtilitiesForTests import get_demo_filepath
src/sorcha/__init__.py:1: in <module>
    from . import modules
src/sorcha/modules/__init__.py:16: in <module>
    from . import PPOutput
src/sorcha/modules/PPOutput.py:10: in <module>
    from tables import NaturalNameWarning
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/tables/__init__.py:44: in <module>
    from .utilsextension import get_hdf5_version as _get_hdf5_version
tables/utilsextension.pyx:1: in init tables.utilsextension
    ???
E   ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
____________ ERROR collecting tests/sorcha/test_sorchaArguments.py _____________
tests/sorcha/test_sorchaArguments.py:3: in <module>
    from sorcha.utilities.sorchaArguments import sorchaArguments
src/sorcha/__init__.py:1: in <module>
    from . import modules
src/sorcha/modules/__init__.py:16: in <module>
    from . import PPOutput
src/sorcha/modules/PPOutput.py:10: in <module>
    from tables import NaturalNameWarning
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/tables/__init__.py:44: in <module>
    from .utilsextension import get_hdf5_version as _get_hdf5_version
tables/utilsextension.pyx:1: in init tables.utilsextension
    ???
E   ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
=========================== short test summary info ============================
ERROR tests/activity/test_activity_registration.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/ephemeris/test_ephemeris_generation.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/ephemeris/test_pixdict.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/ephemeris/test_simulation_driver.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/ephemeris/test_simulation_parsing.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/readers/test_CSVReader.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/readers/test_CombinedDataReader.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/readers/test_DatabaseReader.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/readers/test_HDF5Reader.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/readers/test_OIFReader.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/readers/test_OrbitAuxReader.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_PPApplyFOVFilter.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_PPCheckInputObjectIDs.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_PPCommandLineParser.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_PPConfigParser.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_PPDropObservations.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_PPFootprintFilter.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_PPGetMainFilterAndColourOffsets.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_PPJoinEphemeridesAndOrbits.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_PPJoinEphemeridesAndParameters.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_PPLinkingFilter.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_PPMatchPointingToObservations.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_PPOutput.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_PPRandomizeMeasurements.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_PPReadPointingDatabase.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_combined_data_reading.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_createResultsSQLDatabase.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_demo_end2end.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
ERROR tests/sorcha/test_sorchaArguments.py - ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
!!!!!!!!!!!!!!!!!!! Interrupted: 29 errors during collection !!!!!!!!!!!!!!!!!!!
============================== 29 errors in 3.37s ==============================
Error: Process completed with exit code 2.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -21,7 +21,7 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
-    "numpy",
+    "numpy<2.0",
     "pandas>=2.0",
     "astropy",
     "matplotlib",
@@ -68,7 +68,7 @@ dev = [
     "ipython", # Also used in building notebooks into Sphinx
     "ipykernel",
     "matplotlib", # Used in sample notebook intro_notebook.ipynb
-    "numpy", # Used in sample notebook intro_notebook.ipynb
+    "numpy<2.0", # Used in sample notebook intro_notebook.ipynb
 ]
 
 test = [

